styles/index.css

```diff
@@ -1,48 +1,43 @@
 * {
     box-sizing: border-box;
+    font-family: Helvetica, sans-serif;
```

Comment:
All the text on the page is Helvetica. So why repeat it every time?

Reply:
good point!